include/libtorrent/aux_/session_impl.hpp

```diff
@@ -167,6 +170,8 @@ namespace libtorrent
 		{
 			// the size of each allocation that is chained in the send buffer
 			enum { send_buffer_size_impl = 128 };
+			// maximum length of query names which can be registered by extensions
+			enum { max_dht_query_length = 15 };
```

Comment:
With a size of 15 there is no padding in extention_dht_query for either 32-bit or 64-bit architectures. I want to keep this size of extention_dht_query as small as possible to minimize the number of cache lines m_extension_dht_queries spans.